src/MyEventEmitter.js

```diff
@@ -1,14 +1,71 @@
 'use strict';
 
 class MyEventEmitter {
-  on() {}
-  once() {}
-  off() {}
-  emit() {}
-  prependListener() {}
-  prependOnceListener() {}
-  removeAllListeners() {}
-  listenerCount() {}
+  listeners = {};
+
+  on(event, callback) {
+    if (this.listeners[event]) {
+      this.listeners[event].push(callback);
+    } else {
+      this.listeners[event] = [callback];
+    }
+  }
+
+  once(event, callback) {
+    callback.once = true;
+
+    if (this.listeners[event]) {
+      this.listeners[event].push(callback);
+    } else {
+      this.listeners[event] = [callback];
+    }
+  }
+
+  off(event, callback) {
+    this.listeners[event] = this.listeners[event].filter(
+      (evn) => evn !== callback,
+    );
```

Comment:
The `off` method assumes that `this.listeners[event]` is always defined. If the event does not exist, this will throw an error. Consider adding a check to ensure `this.listeners[event]` is defined before attempting to filter it.